src/Authoring/TestDiagnostics/Scenarios.cs

```diff
@@ -0,0 +1,60 @@
+﻿using ABI.System.Numerics;

```

Comment:
I wouldn't expect you to be using ABI namespaces, you probably want the projected ones in the non ABI namespace?